src/fragmentarium/ui/display/Transliteration.tsx

```diff
@@ -109,32 +112,89 @@ class LineAccumulator {
     )
   }
 }
+function DisplayLineNumberRange({ start, end }: LineNumberRange): JSX.Element {
+  const startPrime = start.hasPrime ? '′' : ''
+  const endPrime = end.hasPrime ? '′' : ''
+  const startPrefixMod = start.prefixModifier ? start.prefixModifier + '+' : ''
+  const startSuffixMod = start.suffixModifier ? start.suffixModifier : ''
+  const endPrefixMod = end.prefixModifier ? end.prefixModifier + '+' : ''
+  const endSuffixMod = end.suffixModifier ? end.suffixModifier : ''
+  const result = `(${startPrefixMod}${start.number}${startPrime}${startSuffixMod}-${endPrefixMod}${end.number}${endPrime}${endSuffixMod})`
+  return (
+    <span>
+      <sup>{result}</sup>
+    </span>
+  )
+}
 
+function DisplayLineNumber({ hasPrime, number }: LineNumber): JSX.Element {
```

Comment:
It is probably not a good idea to use a model type as prop type. See the comment on`DisplayLineNumberRange`.